plugins/clamd.js

```diff
@@ -232,7 +232,7 @@ exports.hook_data_post = function (next, connection) {
 
         var result = '';
         socket.on('line', function (line) {
-            connection.logprotocol(plugin, 'C:' + line);
+            connection.logprotocol(plugin, 'C:' + line.split('').filter(function(x) { var n = x.charCodeAt(0); return 31 < n && 127 > n  }).join('') );
```

Comment:
We can't merge b/c lint tests are failing. We require a space after a function declaration. IE, this change:
```diff
-connection.logprotocol(plugin, 'C:' + line.split('').filter(function(x) { var n = x.charCodeAt(0); return 31 < n && 127 > n  }).join('') );
+connection.logprotocol(plugin, 'C:' + line.split('').filter(function (x) { var n = x.charCodeAt(0); return 31 < n && 127 > n  }).join('') );
```